src/components/Client/Post/Detail/index.jsx

```
@@ -56,148 +57,10 @@ function ContentDetail(props) {
-                                id="form-submit"
-                                className="main-button"
-                              >
-                                Submit
-                              </button>
-                            </fieldset>
-                          </div>
-                        </div>
-                      </form>
+                      <div className="main-content">
+                        <p dangerouslySetInnerHTML={{ __html: post.content }}>
+                        </p>
```

Biome (1.9.4)

[error] Avoid passing both children and the dangerouslySetInnerHTML prop.

This is the source of the children prop

Setting HTML content will inadvertently override any passed children in React

(lint/security/noDangerouslySetInnerHtmlWithChildren)